src/lib/Libwin/win_remote_shell.c

```diff
@@ -114,26 +115,30 @@ disconnect_close_pipe(HANDLE hpipe)
  * @retval	!0 - error number, describing error while trying to create std pipes
  *
  */
-int
+DWORD
```

Comment:
Why are we changing this ?

Reply:
We are changing this because all the 'return' in this function returns the error from GetLastError which has the return type of DWORD. So it is better to have this function's return type as DWORD.